spectator/http_client.cc

```diff
@@ -114,9 +114,9 @@ class LogEntry {
   LogEntry(Registry* registry, std::string method, std::string url)
       : registry_{registry},
         start_{Registry::clock::now()},
-        url_{std::move(url)},
         id_{registry_->CreateId("ipc.client.call",
                                 Tags{{"owner", "spectator-cpp"},
+                                     {"ipc.endpoint", PathFromUrl(url)},
```

Comment:
Extracting the path from the URL can lead to a big metrics explosion if the path can come from user values in any way. For these endpoints, you should get a header back from the server, `Netflix-Endpoint`, that will give you the context. Though, if you get a network failure or something similar it won't be present..